.github/workflows/mlir-tensorrt-ci.yml

```diff
@@ -1,26 +1,39 @@
 name: MLIR-TensorRT CI
 
 on:
-  pull_request:
-    branches:
-      - main
-    types: [synchronize, opened, reopened, ready_for_review]
-    paths: ["mlir-tensorrt/**"]
   push:
+    # this is for PR CI, it will be triggered by PRs via copy-pr-bot
```

Comment:
This might be an issue, how do ensure that PRs isolated to TriPy don't get gated by MLIR-TRT builds? It's wasteful and they probably won't appreciate that...